tslint.json

```diff
@@ -0,0 +1,283 @@
+{
```

Comment:
Would it be "better" to `extend` the base set and then disable the ones that don't apply https://github.com/Microsoft/tslint-microsoft-contrib#which-rules-should-i-turn-on

Reply:
Yes, that seems much cleaner, thanks for the recommendation. I've updated accordingly.